### ISSUE TITLE
List seed locales too

### ISSUE THREAD

**Issue by yvanzo** [houseabsolute/DateTime-Locale#31]
For example “Church Slavic (cu), Volapük (vo), Prussian (prg)” “have been moved in the folder structures” in [CLDR 38](https://cldr.unicode.org/index/downloads/cldr-38). It led to the removal of these locales in `DateTime::Locale` 1.29/1.30.

From the ticket [CLDR-14080](https://unicode-org.atlassian.net/browse/CLDR-14080), it seems that these locales have been moved from “common” to “seed” locales depending on the coverage level.

The opposite move has been made for Sanskrit (sa), making these available in `DateTime::Locale` 1.29/1.30.

It would be useful to list the seed locales in `DateTime::Locale`, even though their level of coverage isn’t as high, rather than having locales to disappear when the coverage requirements change. Seed locales could be listed separately from common locales if the coverage level is important to `DateTime::Locale`.

**Comment by autarch** [comment]
Thanks for the report. I agree that it'd make sense to include these "seed" locales. Any interest in working on a PR to fix this?

**Comment by yvanzo** [comment]
Not at the moment but I still had a quick look at the code:
- There are currently 826 “common” locales for 176 “seed” locales.
- `DateTime-Locale` uses [`cldr-json`](https://github.com/unicode-org/cldr-json/) which doesn’t include “seed” locales from  [`cldr`](https://github.com/unicode-org/cldr/).
  So it is not a small change I hoped it would be.

**Comment by autarch** [comment]
Yeah, I took a look and AFAICT there's no mention of the seed locales in `cldr-json`. Without that I don't think there's any good way to include them. Dealing with the LDML files is an absolute nightmare.
